prototypes/process-explorer/oss-frontend/src/index.html

```diff
@@ -0,0 +1,16 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>processes explorer</title>
+  <base href="/">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="icon" type="image/x-icon" href="favicon.ico">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
```

Comment:
We need material icons, but consume those from an npm package instead of CDN.
See one of our example projects: https://github.com/morganstanley/ComposeUI/blob/main/examples/js-datagrid/package.json#L24C32-L24C32 

For more info: https://github.com/google/material-design-icons --> https://github.com/marella/material-icons#readme

You should import it from the css as the documentation shows.